hangman.js

```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
```

Comment:
How might we refactor these function declarations into methods and attributes on an object?